data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -816,8 +816,9 @@
             [/remove_item]
             [floating_text]
                 x,y=29,31
-                # po: Treat this as if it was a sound-effect in a comic book.
-                text="<span color='#ff7722'>" + _ "Smash!" + "</span>"
+                text="<span color='#ff7722'>" +
+                    # po: Treat this as if it was a sound-effect in a comic book.
+                    _ "Smash!" + "</span>"
```

Comment:
I think the old WML should be acceptable here, but should apply the comment to the translatable string. OTOH, I'd be okay with saying that's a feature that should be postponed to another PR.